### PR TITLE
Update register to vote top actions

### PR DESCRIPTION
**Pull request by hannako**
as per https://docs.google.com/document/d/12feerYNxmQjbPugtY6op38M6ocOBDQHGTu_yoinb2gM/edit#

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by alex-ju — APPROVED** [review]

File: config/machine_readable/register-to-vote.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ preamble: >
 faqs:
   - question: Related content
     answer: >
-      <a href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=actions">Register to vote</a>
-      <a href="https://www.registertovote.service.gov.uk/register-to-vote/start?src=actions">Update your registration</a>
-      <a href="/how-to-vote/postal-voting?src=actions">Apply for a postal vote</a>
+      <a href="/register-to-vote?src=actions">Register to vote</a>
+      <a href="/how-to-vote?src=actions">How to vote</a>
+      <a href="/how-to-vote#voting-by-proxy?src=actions">Ask someone to vote for you</a>
```
Comment (alex-ju) [inline config/machine_readable/register-to-vote.yml:11]:
```suggestion
      <a href="/how-to-vote?src=actions#voting-by-proxy">Ask someone to vote for you</a>
```

File: Rakefile [hunk 2]
```diff
@@ -15,4 +15,4 @@ Rake::TestTask.new(:test_unit) do |t|
 end
 
 Rake::Task[:default].clear if Rake::Task.task_defined?(:default)
-task default: %i[lint test_unit jasmine:ci pact:verify]
+task default: %i[test_unit jasmine:ci pact:verify]
```
Comment (alex-ju) [inline Rakefile:18]: Was the lint removal intentional?

Comment (hannako) [reply Rakefile]: No, sorry! Thank you. Have fixed that.

File: config/machine_readable/register-to-vote.yml (hunk 1 above)
Comment (hannako) [reply config/machine_readable/register-to-vote.yml]: thank you!

**Pull request merged**